api/apiroutes.go

```diff
@@ -52,7 +52,7 @@ type DataSourceLite interface {
 	GetRawTransaction(txid *chainhash.Hash) *apitypes.Tx
 	GetTransactionHex(txid *chainhash.Hash) string
 	GetTrimmedTransaction(txid *chainhash.Hash) *apitypes.TrimmedTx
-	GetRawTransactionWithPrevOutAddresses(txid *chainhash.Hash) (*apitypes.Tx, [][]string)
+	GetRawTransactionWithPrevOutAddresses(txid *chainhash.Hash) (*apitypes.Tx, [][]string, []int64)
```

Comment:
This function is not needed by any consumers of the DataSourceLite interface, but I'm updating it anyway.